### ISSUE TITLE
Service unavailable

### ISSUE THREAD

**Issue by stweil** [jbaiter/demetsiiify#6]
https://demetsiiify.jbaiter.de/ is currently not available (redirected to https://zvdd-ng.de/, then status 404 Not Found).

**Comment by stweil** [comment]
No problem, it's not urgent for me – just a new try to get my favourite historic newspaper shown with IIIF. :-)

**Comment by zuphilip** [comment]
Maybe a heroku app for demetsiiify would also be interesting and independent of any server changes?

**Comment by jbaiter** [comment]
@zuphilip sure, unfortunately I don't have any experience with Heroku :-/ Do they have a free tier that could work? The service needs a recent PostgreSQL database and a Redis instance for the background jobs.

**Comment by zuphilip** [comment]
Yes, they have free option: https://www.heroku.com/pricing and you can AFAIK use docker or docker compose. The databases seems also mentioned in the docu: https://devcenter.heroku.com/categories/data-management . However, I have to admit, that I haven't done myself anything before. @kba did some work at https://github.com/kba/rssscrpr#heroku .

**Comment by kba** [comment]
Yeah, they do support postgres and redis, though not sure whether in the free tier. I have used it for Node.JS and PHP projects but not in the last year and a half which seems an eternity in devops BLANK-as-a-service platform time. Today I would not bother with Procfiles, buildpacks and those heroku-specific technologies and try to get it running with docker as @zuphilip said.

https://github.com/ripienaar/free-for-dev#paas has a nice list of PaaS providers with free tiers.
